site24x7/provider.go

```diff
@@ -50,6 +50,18 @@ func Provider() terraform.ResourceProvider {
 				Default:     4,
 				Description: "Maximum number of retries for Site24x7 API errors until giving up",
 			},
+			"api_base_url": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Default:     "https://www.site24x7.com/api",
```

Comment:
Don't specify a default here, let `site24x7-go` do the defaulting when the `APIBaseURL` config is empty.

Reply:
Default removed.